deployment/modules/gcp/storage/main.tf

```diff
@@ -0,0 +1,66 @@
+terraform {
+  backend "gcs" {}
+
+  required_providers {
+    google = {
+      source  = "registry.terraform.io/hashicorp/google"
+      version = "6.1.0"
+    }
+  }
+}
+
```

Comment:
It might be possible to reference the terraform config for the GCP stuff in the Tessera repo, perhaps something like: 

```
module "tessera-gcs" {
  source = "git@github.com:transparency-dev/trillian-tessera/deployment/module//gcs"
}
```

Then you'd only need to create the dedup DB in here.

Reply:
added a todo